src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/profiledetails/ProfileDetailsSyncService.kt

```diff
@@ -0,0 +1,65 @@
+package uk.gov.justice.digital.hmpps.prisonerfromnomismigration.prisonperson.profiledetails
+
+import com.microsoft.applicationinsights.TelemetryClient
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.helpers.trackEvent
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.prisonperson.ProfileDetailsChangedEvent
+
+@Service
+class ProfileDetailsSyncService(
+  private val nomisApiService: ProfileDetailsNomisApiService,
+  private val dpsApiService: ProfileDetailPhysicalAttributesDpsApiService,
+  private val telemetryClient: TelemetryClient,
+) {
+
+  private fun String.isPhysicalAttributesProfileType() =
+    listOf("BUILD", "L_EYE_C", "R_EYE_C", "HAIR", "FACIAL_HAIR", "SHOESIZE").contains(this)
+
+  suspend fun profileDetailsChanged(event: ProfileDetailsChangedEvent) {
+    val profileType = event.profileType
+    when {
+      profileType.isPhysicalAttributesProfileType() -> profileDetailsPhysicalAttributesChanged(event)
+      else -> {
+        telemetryClient.trackEvent(
+          "profile-details-synchronisation-ignored",
+          mapOf(
+            "offenderNo" to event.offenderIdDisplay,
+            "bookingId" to event.bookingId.toString(),
+            "profileType" to profileType,
+            "reason" to "Profile type not supported",
+          ),
+        )
+      }
```

Comment:
We are only synchronising a subset of the profile details (profile details physical attributes) and the main point of this PR is to ignore the profile types we're not interested in.